src/app/index.config.coffee

```diff
@@ -8,7 +8,7 @@ angular.module 'mnoEnterpriseAngular'
     # Set options third-party lib
     toastrConfig.timeOut = 5000
     toastrConfig.positionClass = 'toast-top-right'
-    toastrConfig.preventDuplicates = true
+    toastrConfig.preventDuplicates = false
```

Comment:
The reason is to prevent duplicates :trollface: 
If you need it for Transac!, we can set it to `false`.